### PR TITLE
[Feat] added responsiveness to result page and changes css

### LINKED ISSUE #17: [FEAT] Add Mobile Responsiveness to frontend

### PR DESCRIPTION
**Pull request by Dhors**
- Added mobile responsiveness to the Result screen by using Bootstrap container, row and col.
- Changed multiple instances of the use of vh to em in the css. 
- Added result pie chart to a popup instead of page.

- Also added new Yumble v2.0 theme styling to Result screen and map modal + result popup.
- Also located bug for session expiry not functioning as intended and removing sessions far too early.

These new changes have been tested by launching the application on a local host and ensuring all functionality is still correct.

New responsive interface in mobile-size screen.
image

Image of pop-up
image

Image of new map modal used across multiple screens (now resizable)
image

Image of new theme matching result screen
image

Image of new theme matching pop-up
image

Image of new screen if no restaurant selected.
image

Closes #17.

### CODE REVIEW

**Review by atha670 — CHANGES_REQUESTED** [review]
Overall the changes are looking  good and should be ready to approve once the merge conflicts have been handled.

**Review by reeved — CHANGES_REQUESTED** [review]
Hi Daniel,
I tried looking into why your build is failing. 
It seems like your merge conflict resolution in `package-lock.json` didn't work properly and you still have artifacts from the resolution process. 

image

If you look at the diff for this file in the "files changed" tab of this PR, you will notice other places in this file where this is repeated.

Please fix this and let me know when it has been resolved :D

**Comment by Dhors** [comment]
> Hi Daniel,
> I tried looking into why your build is failing.
> It seems like your merge conflict resolution in `package-lock.json` didn't work properly and you still have artifacts from the resolution process.
> 
> image
> 
> If you look at the diff for this file in the "files changed" tab of this PR, you will notice other places in this file where this is repeated.
> 
> Please fix this and let me know when it has been resolved :D

It has been solved by pulling the latest branch and running npm install again. Thanks for letting me know Reeve!

**Comment by Dhors** [comment]
> @Dhors Hey Daniel make sure you are changing the merging branch from `main` to `main2` before you merge 😉
> image

Oh my bad, I completely missed that. its now fixed to merging into the right branch of `main2`. Thanks for letting me know David!

**Review by reeved — APPROVED** [review]

**Review by atha670 — APPROVED** [review]

File: frontend/src/components/7.Result/ResultPage.js
Comment (reeved) [inline frontend/src/components/7.Result/ResultPage.js]: could you please change this line to `restaurantLocation={data.coords}`

File: frontend/src/components/7.Result/ResultPage.js
Comment (reeved) [inline frontend/src/components/7.Result/ResultPage.js]: Could you please replace all of this with:
```
                <Card.Header
                  as='h5'
                  id='Card-Header'
                  className='text-center'
                  style={{ width: '100%' }}
                >
                  No Result Decided!
                </Card.Header>
                <Card.Body className='text-center'>
                  <p>Your group was not keen on any restaurants.</p>
                  <Button
                    variant='danger'
                    onClick={() => window.location.reload()}
                  >
                    Try Again{' '}
                  </Button>
                </Card.Body>
```

File: frontend/src/components/7.Result/ResultPage.css
Comment (reeved) [inline frontend/src/components/7.Result/ResultPage.css]: please remove this line. we shouldn't override the font family.

File: frontend/src/components/7.Result/ResultPage.css
Comment (reeved) [inline frontend/src/components/7.Result/ResultPage.css]: We don't need the CSS for .ButtonReset anymore so you can delete all of the CSS related to it on this file.

**Review by reeved — CHANGES_REQUESTED** [review]
Looking good, just would like to see a few changes

File: frontend/src/components/7.Result/ResultPage.js
Comment (Dhors) [reply frontend/src/components/7.Result/ResultPage.js]: done

File: frontend/src/components/7.Result/ResultPage.css
Comment (Dhors) [reply frontend/src/components/7.Result/ResultPage.css]: It has been removed.

File: frontend/src/components/7.Result/ResultPage.css
Comment (Dhors) [reply frontend/src/components/7.Result/ResultPage.css]: Removed, thanks for letting me know that

File: frontend/src/components/7.Result/ResultPage.js
Comment (Dhors) [reply frontend/src/components/7.Result/ResultPage.js]: Done and it looks really clean. Nice changes Reeve

**Review by reeved — APPROVED** [review]

**Comment by reeved** [comment]
Looks great! Nice work Daniel :)

**Pull request merged**
